From: Faisal Latif <faisal.latif@intel.com>
Date: Wed, 20 Jan 2016 13:40:02 -0600
Subject: i40iw: add entry in rdma_netlink
Patch-mainline: v4.6-rc1
Git-commit: 7a43b598a3d60affd13d4082b6afaf09b8131815
References: bsc#1022604

Add entry for port mapper services.

Changes since v2:
	moved this patch before being used

Changes since v1:
	moved I40IW as last element

Signed-off-by: Faisal Latif <faisal.latif@intel.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 include/uapi/rdma/rdma_netlink.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/rdma/rdma_netlink.h b/include/uapi/rdma/rdma_netlink.h
index f7d7b6fec935..6e373d151cad 100644
--- a/include/uapi/rdma/rdma_netlink.h
+++ b/include/uapi/rdma/rdma_netlink.h
@@ -8,6 +8,7 @@ enum {
 	RDMA_NL_IWCM,
 	RDMA_NL_RSVD,
 	RDMA_NL_LS,	/* RDMA Local Services */
+	RDMA_NL_I40IW,
 	RDMA_NL_NUM_CLIENTS
 };
 
-- 
1.8.5.6

